controls/6_1_system_file_permissions.rb

```diff
@@ -191,8 +191,6 @@
   gshadow_files = ['/etc/gshadow']
   gshadow_files << '/usr/share/baselayout/gshadow' if file('/etc/nsswitch.conf').content =~ /^gshadow:\s+(\S+\s+)*usrfiles/
 
-  expected_gid = 0
-  expected_gid = 42 if os.debian?
 
```

Comment:
Can you remove one of the 2 blank lines here ? The checks will pass normally after that.

Reply:
Sorry about that!  

I was trying to make sure the Travis build passed before logging off that day but the build was queued for 30min + and forgot to follow up. 

Would a PR to switch from Travis to GH workflow be something that could happen down the road?

Reply:
We started migrating some of the automation to Github actions (mainly the release process). But there is currently no schedule for the CI.